docs/root/intro/version_history.rst

```diff
@@ -16,6 +16,7 @@ Version history
 * build: releases are built with Clang and linked with LLD.
 * control-plane: management servers can respond with HTTP 304 to indicate that config is up to date for Envoy proxies polling a :ref:`REST API Config Type <envoy_api_field_core.ApiConfigSource.api_type>`
 * csrf: added support for whitelisting additional source origins.
+* dns: upgrade c-ares. Switch from ares_gethostbyname to ares_getaddrinfo.
```

Comment:
This is probably not worth bubbling up to the version history (but your followup PR with the TTL support will definitely belong here).

Reply:
I will remove it.